data/mods/Aftershock/items/tools.json

```diff
@@ -447,5 +447,74 @@
     "symbol": "H",
     "color": "light_gray",
     "flags": [ "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "exotool",
+    "type": "TOOL",
+    "name": { "str": "exoskeleton tool set" },
+    "description": "This is a tool module for a powered exoskeleton.",
+    "weight": "5 kg",
+    "volume": "5 L",
+    "longest_side": "75 cm",
+    "price": 10000,
+    "price_postapoc": 30000,
+    "material": [ "aluminum", "steel" ],
+    "symbol": "=",
+    "color": "light_gray",
+    "use_action": [ [ "CROWBAR", 0 ], [ "HAMMER", 0 ] ],
+    "qualities": [
+      [ "HAMMER", 3 ],
+      [ "HAMMER_FINE", 1 ],
+      [ "SAW_W", 1 ],
+      [ "SAW_M", 2 ],
+      [ "SAW_M_FINE", 1 ],
+      [ "WRENCH", 2 ],
+      [ "WRENCH_FINE", 1 ],
+      [ "WHEEL_FAST", 1 ],
+      [ "SCREW", 1 ],
+      [ "SCREW_FINE", 1 ],
+      [ "CUT", 1 ],
+      [ "PRY", 1 ],
+      [ "PRYING_NAIL", 1 ],
+      [ "DRILL", 3 ],
+      [ "BUTCHER", -38 ]
+    ]
```

Comment:
I dont particularily like this, would be way better to have this be a large  "tool carrier" pocket that lets you fit in a bunch of tools, or alternatively a bunch of non tools at the cost of very minor encumbrance.